### ISSUE TITLE
Bug: no keyboard focus upon start (qt5)

### ISSUE THREAD

**Issue by JonnyTech** [doublecmd/doublecmd#590]
I have just switched from Linux x64 GTK2 builds to QT5 ones.

Upon starting Double Commander, I cannot use the keyboard. The keyboard focus in somewhere unknown. Pressing `tab` does not bring anything into focus. I have to either click the window with my mouse or press `ALT+1` to set the focus and then I can start using it.

This behaviour was not observed previously with gtk2 builds.

```
Double Commander
Version: 1.0.7
Revision: 172
Commit: 0eb417d15
Build date: 2022/09/11
Lazarus: 2.2.2.0
FPC: 3.2.2
Platform: x86_64-Linux-qt5
OS version: MX 21.2 Wildflower

Widgetset library: Qt 5.15.2, libQt5Pas 5.6.1
```

**Comment by alexx2000** [comment]
I cannot reproduce, tested under Xfce and KDE.

**Comment by JonnyTech** [comment]
Thank you for testing. I should have mentioned that I run on xfce4. Further information:

I install from your repo at: https://download.opensuse.org/repositories/home:/Alexx2000/

```
sudo gdebi doublecmd-common_1.0.8-0+svn191~testing_all.deb
sudo gdebi doublecmd-plugins_1.0.8-0+svn191~testing_amd64.deb
sudo gdebi doublecmd-qt_1.0.8-0+svn191~testing_amd64.deb
```

If I install from github artiacts (`doublecmd-1.0.8.qt.x86_64.tar.xz`) then it works correctly.

**Comment by JonnyTech** [comment]
Update: Double v1.0.8 still has the issue.

**Comment by mbetrifork** [comment]
I'm on 1.0.5-beta GTK and I'm having this issue as well. It's very annoying so +1 for a fix for this.

**Comment by JonnyTech** [comment]
I deleted my config folder (`~/.config/doublecmd/`), restarted `doublecmd` and the issue is now gone.

There must have been a setting there causing the issue. If I can recreate the issue then I shall post back here.
